glean-core/android/src/main/java/mozilla/telemetry/glean/private/CounterMetricType.kt

```diff
@@ -22,18 +22,23 @@ import mozilla.telemetry.glean.rust.toBoolean
  *
  * The counter API only exposes the [add] method, which takes care of validating the input
  * data and making sure that limits are enforced.
+ *
+ * The internal constructor is only used by labeled metrics directly.
```

Comment:
nit: you can change this to `"The internal constructor is only used by [LabeledMetricType] directly"` so that the generated docs point to that type.